code/app/build.gradle

```diff
@@ -59,20 +63,15 @@ dependencies {
 
     implementation project(':edgeidentity')
 
-    // TODO: Uncomment next line when core  2.0 artifacts are published to maven
+    // TODO: Update these dependencies when all 2.0 artifacts are published to maven
     // implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
-    implementation 'com.github.adobe.aepsdk-core-android:core:dev-v2.0.0-SNAPSHOT'
-
-    implementation ('com.adobe.marketing.mobile:identity:1+') {
-        transitive = false
-    }
-    implementation('com.adobe.marketing.mobile:edgeconsent:1.+') {
-        transitive = false
-    }
-    implementation('com.adobe.marketing.mobile:edge:1.+') {
+    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
```

Comment:
Can we update `rootProject.mavenCoreVersion` instead?

Reply:
Yes, that would work. Either way we would need to come back and update these dependencies with latest release 2.0.0, so as we discussed let's keep it as is for now.